packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Text/text.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import defaultClasses from './text.css';
+import { mergeClasses } from '../../../../../classify';
+import { arrayOf, shape, string } from 'prop-types';
+
+const toHTML = str => ({ __html: str });
+
+const Text = ({
+    classes,
+    content,
+    textAlign,
+    border,
+    borderColor,
+    borderWidth,
+    borderRadius,
+    marginTop,
+    marginRight,
+    marginBottom,
+    marginLeft,
+    paddingTop,
+    paddingRight,
+    paddingBottom,
+    paddingLeft,
+    cssClasses
+}) => {
+    classes = mergeClasses(defaultClasses, classes);
```

Comment:
This overwrites `props.classes`, which is a bad idea. Consider following the existing pattern:

```es6
const classes = mergeClasses(defaultClasses, props.classes)
```